src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--sun calendar--month-length--30">
+      <div class="calendar__block calendar__block--day--1"></div>
```

Comment:
```suggestion
      <div class="calendar__block"></div>
```

Reply:
it's very complicated to set class with own number for every block
Try to find another way to set positions but without these classes :)